sympy/integrals/integrals.py

```diff
@@ -417,6 +418,15 @@ def doit(self, **hints):
         if self.is_zero:
             return S.Zero
 
+        if isinstance(self.function, Sum):
+            for variable in self.variables:
+                if variable in self.function.limits[0]:
```

Comment:
What if sum has iterated limits?

Reply:
@jksuom I don't completely understand what sum with iterated limits is. Can you give me an example?

Reply:
I was thinking of sums that have multiple indices but it seems that those have not been defined yet, so ignore this question.

But I think that these two lines could be replaced by something like

    if not any(v in self.function.limits for v in self.variables):
        ... compute the integral

Then no `else` part is needed.